tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py

```diff
@@ -0,0 +1,384 @@
+        2. Check ovdataloader type when function get ACTION_DETECTION as task_type
+        3. Check ovdataloader type when function get single video dataset
+    """
+
+    class MockActionOVClsDataLoader(ActionOVClsDataLoader):
+        """Mock class for ActionOVClsDataLoader."""
+
+        def __init__(self, dataset: DatasetEntity, clip_len: int, width: int, height: int):
+            pass
+
+    class MockActionOVDetDataLoader(ActionOVDetDataLoader):
+        """Mock class for ActionOVDetDataLoader."""
+
+        def __init__(self, dataset: DatasetEntity, clip_len: int, width: int, height: int):
+            pass
+
+    mocker.patch("otx.algorithms.action.adapters.openvino.dataloader._is_multi_video", return_value=True)
+    mocker.patch(
+        "otx.algorithms.action.adapters.openvino.dataloader.ActionOVClsDataLoader",
+        side_effect=MockActionOVClsDataLoader,
+    )
+    mocker.patch(
+        "otx.algorithms.action.adapters.openvino.dataloader.ActionOVDetDataLoader",
+        side_effect=MockActionOVDetDataLoader,
+    )
+
+    _task_type = "ACTION_CLASSIFICATION"
+    out = get_ovdataloader(MockDatasetEntity(), _task_type, 8, 256, 256)
+    assert isinstance(out, ActionOVClsDataLoader)
+
+    _task_type = "ACTION_DETECTION"
+    out = get_ovdataloader(MockDatasetEntity(), _task_type, 8, 256, 256)
+    assert isinstance(out, ActionOVDetDataLoader)
+
+    _task_type = "ACTION_SEGMENTATION"
+    with pytest.raises(NotImplementedError):
+        out = get_ovdataloader(MockDatasetEntity, _task_type, 8, 256, 256)
+
+    mocker.patch("otx.algorithms.action.adapters.openvino.dataloader._is_multi_video", return_value=False)
+
+    _task_type = "ACTION_DETECTION"
+    out = get_ovdataloader(MockDatasetEntity(), _task_type, 8, 256, 256)
+    assert isinstance(out, ActionOVDemoDataLoader)
+
+
+@e2e_pytest_unit
+def test_is_multi_video() -> None:
+    """Test _is_multi_video function.
+
+    <Steps>
+        1. Check return value: bool when function get single video DatasetEntity
+        2. Check return value: bool when function get multi video DatasetEntity
+    """
+
+    items: List[DatasetItemEntity] = []
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("2", 0, False))]))
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("2", 1, False))]))
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("2", 2, False))]))
+
+    dataset = DatasetEntity(items)
+    assert _is_multi_video(dataset) is False
+
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("1", 0, False))]))
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("1", 1, False))]))
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("1", 2, False))]))
+
+    dataset = DatasetEntity(items)
+    assert _is_multi_video(dataset) is True
```

Comment:
```suggestion
    assert _is_multi_video(dataset)
```
As far as I know, `is True` make an error during pre-commit.

Reply:
Really? I think == True raise error

Reply:
Sorry. That's my mistake.